### ISSUE TITLE
[2?] Gather tags from nuget.org

### ISSUE THREAD

**Issue by fridex** [openshiftio/openshift.io#810]
# Description

In order to tag Nuget projects, we would like to gather tags from [nuget.org](https://nuget.org). Unfortunately, it looks like nuget.org repository has no API so this gathering needs to be probably done directly on HTML by scraping tags (please check this before starting working on this). Collector should gather all tags present in ecosystem to this date.

It would be also good to parametrize this gathering so we can periodically gather new tags based on project releasing.

# Acceptance criteria

- [ ] Implement tagger collector that would automatically download and collect tags with their occurrence count - more details can be found in [tagger README](https://github.com/fabric8-analytics/fabric8-analytics-tagger/#collecting-keywords---collect), one can also study [already implemented PyPI tags gathering](https://github.com/fabric8-analytics/fabric8-analytics-tagger/blob/master/f8a_tagger/collectors/pypi.py). Tags gathering needs to be fully automated without any additional input (running tagger CLI or directly using already existing tagger library API so this work could be later adopted to other parts e.g. core workers).
- [ ] Tags gathering is run using implemented tagger collector and collected tags are placed in [tags repo](https://github.com/fabric8-analytics/fabric8-analytics-tags) under [raw directory](https://github.com/fabric8-analytics/fabric8-analytics-tags/tree/master/raw), respecting tagger tags format
- [ ] License of nuget.org data (tags) is checked so tags can be used
- [ ] Check for possible other ways on how to aggregate these tags (API?)

**Comment by fridex** [comment]
Similar to https://github.com/openshiftio/openshift.io/issues/712, https://github.com/openshiftio/openshift.io/issues/710 and https://github.com/openshiftio/openshift.io/issues/725.

**Comment by miteshvp** [comment]
No longer relevant.
